lib/Common.ts

```diff
@@ -329,6 +329,21 @@ export interface GridColumn<T> {
     width?: number;
 }
 
+const setRef = <T>(ref: React.Ref<T>, value: T) => {
```

Comment:
I was two concerns with this approach:
1. This amount of change here seems overkill just to get a ref propagated through lodash :)
2. How do you know which element to apply the ref to? The attributes API for these components goes through great lengths to disambiguate between containers/inputs/balloons etc., it's a bit weird if we have an adjacent API that ignores all this and just hardcodes to the input element

I would suggest trying one of these approaches instead:
1. immer
2. Specify autocomplete in explorer's 
3. Don't use refs, just use ids/classnames/attributes to access the htmlelement

Reply:
I can see how this could be an overkill for the original scenario. But for your 2nd point, as the react docs say on forwardingRefs, when creating a low level controls library, most cases you want to obfuscate the access to the DOM element, but with buttons or fields, most of the time you want to give access to that specific element, since they are always present and the controls library should work as a wrapper around the specific field or button.

To fix the explorer scenario I can just use ids/attributes (since we do this same thing in other places), but I would suggest we think about supporting forwarding refs through a similar approach to the one suggested in this PR